src/easydiffraction/__init__.py

```diff
@@ -9,7 +9,8 @@
 from .components.phase import Phases as Phases
 from .components.site import Atoms as Atoms
 from .components.site import Site as Site
-from .io.download import download_from_repository
-from .Job import DiffractionJob as Job
+from .job import DiffractionJob as Job
+from .utils import download_from_repository
 
-__all__ = ['Job', 'download_from_repository']
+__all__ = ['Job',
+           'download_from_repository']
```

Comment:
there is no reason to break this line, it's only two elements

Reply:
done